.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
.vscode folder is specific to vscode IDE, so usually these kind of folders are added to .gitignore so that their changes are not visible.
However, as I understand, our whole team is using vscode IDE, so maybe it is ok? Open for discussion